### PR TITLE
feat(feature dev): update more customer facing texts and doc link

### PR DESCRIPTION
**Pull request by willyyhuang**
## Problem

We observed a couple customer copy papercuts related to /dev

## Solution

- Updated customer facing text copies in /dev 
- Changed documentation link
- Removed a close session messaging when clicking for new task

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/amazonq/webview/ui/tabs/constants.ts
Comment (jpinkney-aws) [inline packages/core/src/amazonq/webview/ui/tabs/constants.ts:28]: Please make sure this welcome message is formatted correctly when you load up the extension. I've had to fix those issues so many times 😅

**Review by jpinkney-aws — APPROVED** [review]
LGTM, just waiting for approval from your team. Have you tested this PR by running the actual extension? I've seen it happen a few times were people didn't test and the formatting gets thrown off. Also, has this been approved by a UX designer/doc writer/pm, etc?

File: packages/core/src/amazonq/webview/ui/tabs/constants.ts
Comment (willyyhuang) [reply packages/core/src/amazonq/webview/ui/tabs/constants.ts:28]: image

**Review by osdemah — APPROVED** [review]
LGTM

File: packages/core/src/amazonq/webview/ui/tabs/constants.ts
Comment (neilk-aws) [inline packages/core/src/amazonq/webview/ui/tabs/constants.ts:31]: I think this is a typo. "What would you like _to_ work on?"

**Review by neilk-aws — CHANGES_REQUESTED** [review]

**Review by neilk-aws — APPROVED** [review]

**Pull request merged**
